.github/workflows/self_tests.yml

```
@@ -2,10 +2,20 @@ name: GLuaTest (Self Tests)
 
 on:
   pull_request:
+  push:
```

We can add push triggers here, but only so long as it's to `main`
```suggestion
  push:
    branches:
      - 'main'
```